src/ui-lib/sass/03-elements/_section.scss

```diff
@@ -4,4 +4,9 @@ section {
   + section {
     margin-top: $grav-sp-xl;
   }
+  
 }
+.section--background-banner {
```

Comment:
this should go in a more appropriate place, as by the look of it, it can be applied to anything. Perhaps as a utility class?